packages/wrangler/src/dev/miniflare.ts

```diff
@@ -68,6 +68,7 @@ const HANDLER_RESERVED_KEYS = new Set([
 	"webSocketClose",
 	"webSocketError",
 	"self",
+	"tailStream"
```

Comment:
nit: trailing comma, order alphabetically.

Thanks for taking care of the update ❤️